### DISCUSSION TITLE
What version of aws-sdk-java does CVE-2021-4277 affect?

### DISCUSSION THREAD

**Discussion by nefnohack** [aws/aws-sdk-java#2948] [Q&A]
CVE-2021-4277

https://github.com/jeremylong/DependencyCheck/actions/runs/3843398265

**Comment by BruceLuX** [comment]
We also have the same question, any one can help ?

**Comment by debora-ito** [comment]
According to [CVE-2021-4277](https://github.com/advisories/GHSA-3cqm-26w8-85g8): "A vulnerability, which was classified as problematic, has been found in fredsmith utils". 

The Java SDK doesn't use fredsmith utils, and the link to the [commit](https://github.com/fredsmith/utils/commit/dbab1b66955eeb3d76b34612b358307f5c4e3944) gives me a 404, so I think the repository is private.

Can you elaborate on how do you think the security report is related to the Java SDK?
